simulation/executor/internal/stats/stats_db.go

```diff
@@ -55,7 +55,10 @@ func (stats StatsDb) LogActionResult(header tmproto.Header, opMsg simulation.Ope
 	}
 	appHash := fmt.Sprintf("%X", header.AppHash)
 	resData := fmt.Sprintf("%X", resultData)
+	// UNFORKINGTODO OQ: Is GasWanted and GasUsed getting added? If so, add and uncomment
+	// If not, remove from INSERT statement below
```

Comment:
We should be able to still unpack this I think, but can punt for later

Reply:
changed to unforking note https://github.com/osmosis-labs/osmosis/pull/6413/commits/2e1c779ac8cb2bf84cc0b1f1c9d0faaffc7d30aa